### PR TITLE
Gitlab support for review-rot

### PR DESCRIPTION
**Pull request by nirzari**
Something went wrong while rebasing. This should work fine.
I apologize for the inconvenience.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
       install_requires=[
           'PyGithub',
           'PyYAML',
-          'dateutils',
```
Comment (ralphbean) [inline setup.py]: Ah, did you mean to delete `'dateutils'` here?

Comment (nirzari) [inline setup.py]: Opss, that must have happened while the merge issues I had while rebasing stuff.
I will re-add that.

Comment (nirzari) [inline setup.py]: @ralphbean I tried re adding that and I am getting this 'raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'dateutils' distribution was not found and is required by review-rot'

**Comment by ralphbean** [comment]
Ah, after that you have to run `python setup.py develop` again.

**Comment by nirzari** [comment]
Of course , I surely did `python setup.py develop`. I was installing 'dateutil' instead of 'dateutils' and Hence the error.
However I am not so sure why do we need 'dateutils'.

**Comment by ralphbean** [comment]
For `from dateutil.relativedelta import relativedelta` in githubstack.

**Comment by ralphbean** [comment]
Amazingly, the `dateutils` package provides the `dateutil` module (notice the difference).

**Pull request merged**